src/test/java/pl/jakubkozlowski/learning/firststeps/controller/ChampionControllerTest.java

```diff
@@ -75,7 +75,7 @@ public void whenFindAll_thenReturnJsonArrayOfChampions() throws Exception {
     @Test
     public void whenFindOne_thenReturnJsonArrayOfChampionWithGivenID() throws Exception {
 
-        String content = mvc.perform(get(BASE_PATH + ID_1)
+        String content = mvc.perform(get(BASE_PATH + BY_ID)
```

Comment:
I don't think that test would pass. You should call exact id not basepath/{id} right?